scripts/zip_gcs_files.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -ex
```

Comment:
`-x` here means that each command will be logged as it is executed, so we don't need any explicit `echo` commands for logging.